application/src/main/java/org/togetherjava/tjbot/features/github/GitHubReference.java

```diff
@@ -43,8 +43,14 @@ public final class GitHubReference extends MessageReceiverAdapter {
      */
     static final Pattern ISSUE_REFERENCE_PATTERN =
             Pattern.compile("#(?<%s>\\d{1,5})".formatted(ID_GROUP));
-    private static final int ISSUE_OPEN = Color.green.getRGB();
-    private static final int ISSUE_CLOSE = Color.red.getRGB();
+
+    // Representing different GitHub states of an Issue/PR
+    private static final int OPEN_STATE = Color.green.getRGB();
+    private static final int CLOSE_STATE = Color.red.getRGB();
+    private static final int MERGED_STATE = new Color(141, 106, 187).getRGB();
+    private static final int NOT_PLANNED_STATE = new Color(72, 72, 72).getRGB();
+    private static final int DRAFT_STATE = Color.gray.getRGB();
+
```

Comment:
why are these `int` instead of `Color`

Reply:
It was already int before, so i kept it like that. However, it does make sense to change it to **"Color"**